app/views/content/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers.md

```diff
@@ -42,27 +42,34 @@ calls_to_action:
     arguments:
       - Cost: "This could be around $earlyyears_fees_selffundedlow$ or $earlyyears_fees_selffundedhigh$ depending on the provider"
         Salary: "No"
         Length of training: "3 months"
         Qualification: "Early years teacher status"
+  EY-apprenticeship:
+    name: feature_table
+    arguments:
+      - Cost: "Your course fees of up to $earlyyears_fees_apprenticeship$ per year are covered by your employer’s apprenticeship levy payments or co-funded between your employer and the Department for Education. You do not need to do anything to access this funding."
+        Salary: "Provided by employer"
+        Length of training: "3 years"
+        Qualification: "Early years teacher status, a bachelor’s degree, and level 6 early years teacher apprenticeship"
 ---
 
 The early years are critical in children’s development. As an early years teacher you’ll work with young children up to the age of 5. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
 
-You’ll need to do early years initial teacher training (EYITT) to achieve early years teacher status (EYTS). This will allow you to work in a nursery or early years setting.
+To teach in a nursery or early years setting, you'll need early years teacher status (EYTS). EYTS is also recognised in academies, free schools and independent schools. You can achieve EYTS through either early years initial teacher training (EYITT) or an early years teacher degree apprenticeship (EYTDA).
 
-If you want to teach in a primary school, including in a reception class, or secondary school then you will need to achieve qualified teacher status (QTS). [Find out more about training to teach primary and secondary pupils](/train-to-be-a-teacher).
+If you want to teach in a primary school, including in a reception class, then you'll need to achieve qualified teacher status (QTS). [Find out more about training to teach primary pupils](/life-as-a-teacher/age-groups-and-specialisms/primary).
 
 If you're interested in exploring other opportunities to work with pre-school children, you can [find out more about a range of careers in early years education](https://earlyyearscareers.campaign.gov.uk/?utm_source=Website&utm_medium=GiT). 
 
 ## What you need to become an early years teacher
 
-There are 2 ways to train to become an early years teacher, either as a postgraduate if you already have a degree, or as part of an undergraduate degree.
+There are several ways to train to become an early years teacher. These routes include undergraduate degree training, an early years teacher degree apprenticeship, or postgraduate options if you already have a degree. All of these routes lead to early years teacher status (EYTS).
 
 To do postgraduate training, you’ll need a bachelor’s degree or an equivalent, for example, a vocational or professional qualification, or equivalent qualification you gained outside the UK.
```

Comment:
@emmaachesongray I wonder if it makes more sense to put this postgrad para after the GCSEs (I know this is in prod already)

ie for all routes you need GCSEs, for postgrad you'll **also** need a degree

(if that is how it works!)